hexgame/game.py

```diff
@@ -27,8 +27,30 @@ def __init__(
         self.current_player: Player = self.player_1
 
     def _play(self) -> None:
-        # player  move
-        self.current_player.play(self.board)
+        moved = False
+        trials_cap = 10
```

Comment:
Magic numbers are bad, let's put a constant or in a config file, together with things like board size